awesome_dashboard/static/src/dashboard/dashboard.js

```diff
@@ -33,6 +37,19 @@ class AwesomeDashboard extends Component {
             ],
         });
     }
+
+    updateConfig(newDisabledItems) {
+        this.disabledItems.values = newDisabledItems;
+        browser.localStorage.setItem("disabledDashboardItems", newDisabledItems);
```

Comment:
we don't need to use browser here
```suggestion
        localStorage.setItem("disabledDashboardItems", newDisabledItems);
```